packages/core/src/template-compiler.ts

```diff
@@ -37,9 +37,15 @@ interface GlimmerSyntax {
   print(ast: AST): string;
   defaultOptions(options: PreprocessOptions): PreprocessOptions;
   registerPlugin(type: string, plugin: unknown): void;
-  precompile(templateContents: string, options: {
-    contents: string; moduleName: string; filename: string, plugins?: any
-  }): string;
+  precompile(
+    templateContents: string,
+    options: {
```

Comment:
FWIW, it also accepts `parseOptions?: { srcName?: string }`

https://github.com/ember-cli/ember-cli-htmlbars/blob/4fdc8949917a3f79a33a038fa581f183cf74bd1d/lib/template-compiler-plugin.js#L75-L77

Reply:
Thanks for catching that, I'll add this as well.